ckanext/unaids/dataset_transfer/logic.py

```diff
@@ -60,16 +60,16 @@ def send_dataset_transfer_emails(dataset_id, recipient_org_id):
             DatasetTransferRequest.dataset_id == dataset_id,
             DatasetTransferRequest.recipient_org_id == recipient_org_id
         ).all()
-    users_to_email = _get_users_to_email(
-        recipient_org=recipient_org,
+    users_to_email = get_org_admins_with_email_addresses(
+        org=recipient_org,
         exclude_user_ids=user_ids_already_emailed
     )
 
     for user in users_to_email:
         try:
-            subject = 'testing123'
+            subject = _('Dataset Transfer')
```

Comment:
Could you include the title of the dataset in the e-mail subject. That should be more informative.

Reply:
Sure, i'll make it `<DATASET_TITLE> Dataset Transfer`